lib/css/components/select.less

```diff
@@ -0,0 +1,194 @@
+    }
+
+    .has-error & {
+        --_se-select-bc: var(--red-400);
+        --_se-select-bc-focus: var(--red-400);
+        --_se-select-bs-focus: 0 0 0 var(--su-static4) var(--focus-ring-error);
+    }
+
+    .has-success & {
+        --_se-select-bc: var(--green-400);
+        --_se-select-bc-focus: var(--green-400);
+        --_se-select-bs-focus: 0 0 0 var(--su-static4) var(--focus-ring-success);
+    }
+
+    .has-warning & {
+        --_se-select-bc: var(--yellow-600);
+        --_se-select-bc-focus: var(--yellow-600);
+        --_se-select-bs-focus: 0 0 0 var(--su-static4) var(--focus-ring-warning);
+    }
+
+    // CHILD ELEMENTS
+    &,
+    & > select {
+        &::-webkit-contacts-auto-fill-button {
+            background-color: var(--black); // In Safari, make the autocomplete button darkmode-aware
+        }
+
+        &:-webkit-autofill {
+            border-color: var(--blue-300);
+            -webkit-text-fill-color: var(--black);
+            -webkit-box-shadow: 0 0 0 1000px var(--theme-secondary-050) inset; // This acts as a background color by stretching an inset box shadow across the input
+            transition: background-color 0s 50000s; // A hack to infinitely delay background styles that come from the browser.
+
+            &:focus {
+                border-color: var(--blue-300);
+                // Since the box shadow is overwritten to show a background, we have to re-add the focus outline
+                -webkit-box-shadow: 0 0 0 1000px var(--blue-050) inset, 0 0 0 var(--su-static4) var(--focus-ring);
+            }
+        }
+    }
+
+    &:before,
+    &:after { // menu arrows
+        border-color: var(--_se-arrow-bc);
+        border-style: solid;
+        border-width: var(--_se-arrow-size);
+        content: "";
+        pointer-events: none;
+        position: absolute;
+        right: calc(var(--su-static12) + var(--su-static1));
+        z-index: var(--zi-selected);
+    }
+
+    &:after {
+        border-bottom-width: 0;
+        top: calc(50% + var(--su-static1));
+    }
+
+    &:before {
+        border-top-width: 0;
+        top: calc(50% - calc(var(--_se-arrow-size) + var(--su-static1)));
+    }
+
+    > select { // Menu
+        // CONTEXTUAL STYLES
+        fieldset[disabled] &,
+        &[disabled] {
+            cursor: not-allowed;
+            opacity: var(--_o-disabled-static);
+        }
+
+        &[readonly],
+        .is-readonly & { // [1]
+            --_se-select-bc: var(--bc-light);
+            --_se-select-bg: var(--black-050);
+            --_se-select-fc: var(--black-200);
+
+            .highcontrast-mode({
+                --_se-select-fc: var(--fc-light);
+            });
+
+            cursor: not-allowed;
+        }
+
+        // CHILD ELEMENTS
+        &::-moz-focus-inner {
+            outline: none !important;
+        }
+
+        &:-moz-focusring {
+            color: transparent;
+            text-shadow: 0 0 0 #000;
+        }
+
+        &::-ms-expand {
```

Comment:
Double check - can almost assuredly remove (Edge Legacy)

Reply:
I'm removing it. It's hard to _even find_ documentation on it, so I'm pretty we can trash it. Best docs I found https://udn.realityripple.com/docs/Archive/Web/CSS/::-ms-expand